src/schema/nmdc.yaml

```diff
@@ -617,6 +618,15 @@ classes:
     class_uri: prov:Association
 
 enums:
+  biosample_category_enum:
+    description: Categories that a biosmaple can belong to, whether funding-based, methods-based, etc.
```

Comment:
For description, I might suggest revising to: "Defined categories to designate funding-based, sample location-based, or experimental method-based attributes to enable metadata search or filtering. For example, SFA (Science Focus Area project funded through the Department of Energy Office of Science Biological and Environmental Research Program) is a category tag."

Reply:
I think I got all of your ideas in. Let me know if you can't see what I'm taking about on the Files Changed tab